samples/logging-sample/README.md

```diff
@@ -0,0 +1,43 @@
+# Cloud Run System Package Sample
+
+This sample shows how to create a simple Cloud Build notifier.
+
+Use it with the [Creating your own notifer tutorial](https://cloud.google.com/cloud-build/docs/configuring-notifications/create-notifier).
+
+## Upload the config to a Google Cloud Storage bucket 
+
+First create a bucket.
+
+```sh
+export BUCKET_NAME=your_bucket_name
+gsutil mb gs://$BUCKET_NAME/
+```
+
+Then upload the provided notifier config to the bucket.
+
+```sh
+export CONFIG_FILE_NAME=notifier_config.yaml
+gsutil cp $CONFIG_FILE_NAME gs://$BUCKET_NAME/$CONFIG_FILE_NAME
+```
+
+## Build and Deploy
+
+```sh
+gcloud builds submit .  --substitutions=_CONFIG_PATH=gs://$BUCKET_NAME/$CONFIG_FILE_NAME
```

Comment:
Up to you: state that this command should be run in this `samples/logging-sample` directory?